tests/tests_pandas_excel/test_pandas_excel_reader.py

```diff
@@ -17,4 +17,4 @@ def test_bad_sheet():
     # file = open("bad_sheet.xlsx", "r")
     
     # load the bad xlsx sheet
-    pandas_excel_reader.load_data(file=Path('tests/tests_pandas_excel/bad_sheet.xlsx'))
\ No newline at end of file
+    pandas_excel_reader.load_data(file=Path('tests/tests_pandas_excel/alec_test.xlsx'))
```

Comment:
please leave the original name and just replace the bad_sheet.xlsx file with my file.  That's better than checking in a file with my name on it.